src/backend/launcher.ts

```diff
@@ -1365,10 +1365,24 @@ async function runAfterLaunchScript(
  */
 async function runScriptForGame(
   gameInfo: GameInfo,
-  scriptPath: string
+  gameSettings: GameSettings,
+  scriptStage: 'before' | 'after'
 ): Promise<boolean | string> {
   return new Promise((resolve, reject) => {
-    const child = spawn(scriptPath, { cwd: gameInfo.install.install_path })
+    const scriptPath = gameSettings[`${scriptStage}LaunchScriptPath`]
+    const scriptEnv = {
+      HEROIC_GAME_APP_NAME: gameInfo.app_name,
```

Comment:
adding `...process.env` here would eliminate the need for the object.assign later is cleaner imo